wsmaster/che-core-api-devfile/src/main/resources/build_docs.sh

```diff
@@ -112,6 +118,7 @@ print_help() {
    echo "   --no-deploy   Skip deploy result to remote"
    echo "   --org         Specifies organization of devfile repository to which Docs should be pushed. Default value: redhat-developer"
    echo "   --branch|-b   Specifies branch to which Docs should be pushed. Default behaviour: use default GitHub repo branch"
```

Comment:
Default behaviour is changed and now default branch is `docs_renewal`, right?

Reply:
yes